libhb/cropscale.c

```diff
@@ -194,7 +194,8 @@ static hb_buffer_t* crop_scale( hb_filter_private_t * pv, hb_buffer_t * in )
                                 in->f.width  - (pv->crop[2] + pv->crop[3]),
                                 in->f.height - (pv->crop[0] + pv->crop[1]),
                                 in->f.fmt, out->f.width, out->f.height,
-                                out->f.fmt, SWS_LANCZOS|SWS_ACCURATE_RND);
+                                out->f.fmt, SWS_LANCZOS|SWS_ACCURATE_RND,
+                                SWS_CS_DEFAULT);
```

Comment:
It could be pretty much any that are supported by any of the libav decoders. I.e. whatever AV_PIX_FMT_* that the source video is using. So quite a variety.